src/Components/dashboard/operatorsanddriver/DriverAndOperator.jsx

```diff
@@ -11,7 +11,7 @@ import Box from "@mui/material/Box";
 import Tab from "@mui/material/Tab";
 import TabContext from "@mui/lab/TabContext";
 import TabList from "@mui/lab/TabList";
-import Skeleton from "react-loading-skeleton";
+
```

Comment:
and this too

Reply:
This one was not yet implemented, will take care during its implementation.